app/views/forums/show.html.erb

```diff
@@ -1,8 +1,15 @@
 <%= link_to @forum.group, forumgroup_path(@forum.group) %> → <%= @forum %>
 
-<h1><%= title @forum %></h1>
+<h1>
+  <%= title @forum %>
+  <% params[:id] = params[:id].split("-")[0] %>
```

Comment:
When going to the threads view from the forum, it would pass the id param as ?id=#-name.